src/components/PeopleManagement/data/hooks/useEnterpriseMembersTableData.js

```diff
@@ -34,7 +35,13 @@ const useEnterpriseMembersTableData = ({ enterpriseId }) => {
             options.user_query = value;
           }
         });
-
+        if (args?.sortBy.length > 0) {
```

Comment:
Can we add tests for sorting to [useEnterpriseMembersTableData.test.jsx](https://github.com/openedx/frontend-app-admin-portal/blob/3428c0088416150d6b66b4ef23439a382c20e753/src/components/PeopleManagement/tests/useEnterpriseMembersTableData.test.jsx#L7) ?